core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -125,6 +122,26 @@ private CompletionSuggestionBuilder(String fieldname, CompletionSuggestionBuilde
     }
 
     /**
+     * Read from a stream.
+     */
+    public CompletionSuggestionBuilder(StreamInput in) throws IOException {
+        super(in);
+        payloadFields = new ArrayList<>();
```

Comment:
I kinda thought we preferred Lists for builders because they are more Java-ish? I don't mind either way.